bliss/simulator/prior.py

```diff
@@ -216,7 +254,8 @@ def _sample_galaxy_prior(self):
             self.galaxy_alpha, self.galaxy_flux_min, self.galaxy_flux_max, latent_dims
         )
 
-        total_flux[:, :, :, :, 1:] = self.multiflux(3, total_flux[:, :, :, :, 0])
+        if self.n_bands > 1:
```

Comment:
Condition checks if the number of bands is greater than 1 because if only single-band inference is requested, there is no need for any sampling from the SDSS catalog.